synapse/rest/media/thumbnail_resource.py

```diff
@@ -155,30 +155,24 @@ async def _select_or_generate_local_thumbnail(
 
         thumbnail_infos = await self.store.get_local_media_thumbnails(media_id)
         for info in thumbnail_infos:
-            t_w = info["thumbnail_width"] == desired_width
-            t_h = info["thumbnail_height"] == desired_height
-            t_method = info["thumbnail_method"] == desired_method
-            t_type = info["thumbnail_type"] == desired_type
+            t_w = info.width == desired_width
+            t_h = info.height == desired_height
+            t_method = info.method == desired_method
+            t_type = info.type == desired_type
```

Comment:
Aside: not sure what `t_` stands for here. True? Test?

Reply:
I have no idea TBH. 🤷 "test" would be my guess?